services/twilio-mock-service/package.json

```
@@ -0,0 +1,51 @@
+{
+  "name": "twilio-mock-service",
+  "author": "121.global",
+  "homepage": "https://www.121.global/",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/global-121/121-platform.git",
+    "directory": "services/twilio-mock-service"
+  },
+  "main": "index.js",
+  "scripts": {
+    "lint": "npm run lint:code && npm run lint:syntax",
+    "lint:code": "eslint \"src/**/*.{ts,js}\"",
+    "fix:code": "npm run lint:code -- --fix",
+    "lint:syntax": "prettier --check \"**/*.{md,ts,js,json,yml}\"",
+    "fix:syntax": "npm run lint:code -- --fix && npm run lint:syntax -- --write",
+    "start": "GLOBAL_121_VERSION=$(test -f \"VERSION.txt\" && cat VERSION.txt || echo $GLOBAL_121_VERSION)  node index.js",
```

```suggestion
    "start": "node index.js",
```

All 121-service specific customizations are not necessary for the Twilio Mock Service, right?